src/content/blog/2025-05-05-celebrating-ten-years-of-discover.mdx

```diff
@@ -0,0 +1,68 @@
+
+## The Enduring Impact of Publicly Available Imagery
+
+As of the writing of this article, Discover has seen over 5.4 billion requests over its lifetime. Each of these requests represents a pan or zoom made by a wide variety of users, from government agencies to engineering firms to data managers. These entities use the services for all kinds of workflows, including many that impact people in Utah every single day.
+
+For example, the [Utah Division of Emergency Management](https://dem.utah.gov/) uses aerial imagery to safeguard Utah's future and help mitigate the effects of natural disasters in our state. In their words:
+
+"The Division and all our partners in every jurisdiction at every level, rely on high-resolution imagery to respond to, recover from, and mitigate disaster that can affect every single citizen of Utah.
+
+Everyone, regardless of whether they live in an urban or rural area, deserves the best response possible from the division which necessitates having the best information possible to do our jobs."
+
+The [Utah Department of Transportation](https://www.udot.utah.gov/connect/) also uses aerial imagery in their daily work, helping every one of us get to where we need to go. Corey Unger, the spatial technologies manager at UDOT, had this to say about Discover:
+
+“High quality aerial imagery, like that provided by UGRC is a valuable resource for various UDOT departments. It aids in fatal crash reviews, design processes, and environmental surveys. Designers utilize it for basemaps and asset location verification, while the LRS department relies on it for updating road information. The 6-inch resolution imagery is particularly preferred for its accuracy and detail, crucial for tasks like ADA ramp placement and road design.”
+
+We are proud to provide the services for these incredible agencies, as well as many others. Throughout the state, all kinds of decision makers are using aerial imagery for:
+
+- Water budgeting and distribution
+- Rural public land issues
+- 811 Call-Before-You-Dig center
+- Broadband access expansion
+- 911 and emergency response
+- Natural resources management
+- Infrastructure/Asset Management
+
+The list could go on and on!
+
+The wide variety of use cases for Discover is truly what's worth celebrating. These services wouldn't be worth much if they just sat around collecting dust. But because of the amazing people we've had the joy of working with, those service become more than just pixels on a screen. Over the last ten years, we have had the opportunity to see these products transform lives and bolster data-driven decision making in our state. From the background processes that underpin everyday life, to the emergency management practices made possible through well-maintained data, Discover is just one way we “encourage and facilitate the effective use of geospatial information and technology for Utah.”
```

Comment:
```suggestion
The wide variety of use cases for Discover is truly what's worth celebrating. The basemaps and imagery wouldn't be worth much if they just sat around collecting dust. But, because of the amazing people we've had the joy of working with, those service become more than just pixels on a screen. Over the last ten years, we have had the opportunity to see these products transform lives and bolster data-driven decision making in our state. From the background processes that underpin everyday life, to the emergency management practices made possible through well-maintained data, Discover is just one way we “encourage and facilitate the effective use of geospatial information and technology for Utah.”
```
Trying to emphasize the content, not just the delivery platform.